carla:
    use_offset: true

carla_vehicle_interface:
    max_steer_angle: 70.0
